oggm/utils.py

```diff
@@ -3161,10 +3161,10 @@ def shape_factor_huss(widths, heights, is_rectangular):
 
     # Ensure bool (for masking)
     is_rectangular = is_rectangular.astype(bool)
```

Comment:
Avoid typecast necessary? Did not look into the implementation and cannot say anything about perfomance impacts